docs/integrations/sources/chartmogul.md

```diff
@@ -1,46 +1,61 @@
 # Chartmogul
-This page contains the setup guide and reference information for the Chartmogul source connector.
+This page contains the setup guide and reference information for the [Chartmogul](https://chartmogul.com/) source connector.
 
 ## Prerequisites
-* API key
-* Start date
-* Interval
+- A Chartmogul API Key.
+- A desired start date from which to begin replicating data.
+- A desired interval period for the `CustomerCount` stream. The available options are **day**, **week**, **month**, and **quarter**.
 
 ## Setup guide
-### Step 1: Set up Chartmogul
-
-1. To get access to the Chartmogul API you need to create an API key, please follow the instructions in this [documentation](https://help.chartmogul.com/hc/en-us/articles/4407796325906-Creating-and-Managing-API-keys#creating-an-api-key).
+### Step 1: Set up a Chartmogul API key
+1. Log in to your Chartmogul account.
+2. In the left navbar, select **Profile** > **View Profile**.
+3. Select **NEW API KEY**.
+4. In the **Name** field, enter a unique name for the key.
+5. If you are a Staff, Admin, or Owner, set the **Access Level** to either **Read-only** or **Read & Write** using the dropdown menu.
```

Comment:
By [principle of least privilege ](https://csrc.nist.gov/glossary/term/least_privilege#:~:text=Definition(s)%3A,needs%20to%20perform%20its%20function.)
```suggestion
5. If you are a Staff, Admin, or Owner, set the **Access Level** to either **Read-only** or **Read & Write** using the dropdown menu. We recommend **Read-only**. 
```

Reply:
Added, and I learned a new principle to boot! Thanks for sharing the link